sass/pages/_assets.scss

```
@@ -18,29 +18,109 @@
         margin-bottom: 52px;
     }
 
-    .card {
-        margin: 0;
-        height: 8rem;
-    }
+    .asset-card {
+        display: grid;
+        grid-template-areas:
+            "banner"
+            "desc"
+            "tags";
+        grid-template-rows: 12rem 5.5rem 2rem;
 
-    .card-title {
-        font-size: 1.3rem;
-        overflow-wrap: break-word;
-    }
+        text-decoration: none;
+        overflow: hidden;
+        border-radius: $border-radius;
+        background-color: $syntax-theme-background;
+        color: $color-white;
+        border: 2px solid #383838;
 
-    .card-description {
-        font-size: 1rem;
-        height: initial;
-    }
+        &:hover {
+            background-color: $card-hover-background;
+            border: 2px solid #6b6b6b;
+        }
 
-    .card-image > img {
-        display: block;
-        height: auto;
-        max-width: 100%;
-    }
+        &__banner {
+            grid-area: banner;
+            display: block;
+            position: relative;
 
-    a {
-        text-decoration: none;
-        color: $link-color;
+            background: linear-gradient(45deg,
+                    $default-image-background-color 0%,
+                    $card-hover-background 40%,
+                    $default-image-background-color 100%);
+
+            img {
+                width: 100%;
+                height: 100%;
+                object-fit: contain;
+
+                background-color: $default-image-background-color;
+            }
+        }
+
+        &__title {
+            position: absolute;
+            width: 100%;
+            bottom: 0;
+
+            padding: 0.6rem;
+            font-weight: bold;
+            font-size: 1.2rem;
+            background-color: rgba(0, 0, 0, 0.5);
+        }
+
+        &__description {
+            grid-area: desc;
+            display: flex;
+
+            padding: 0.6rem;
+            font-size: 1rem;
+        }
```

I've noticed that this doesn't handle extra long text correctly.

**Option A**: use 4 lines ellipsis ([the mixin uses this snippet](https://css-tricks.com/almanac/properties/l/line-clamp/)).

```suggestion
        &__description {
            @include text-ellipsis(4);

            grid-area: desc;

            padding: 0.6rem;
            font-size: 1rem;
        }
```

**Option B**, see comment:

---

Sorry, I didn't realize, I saw it a after you removed it again on the comment. 🤦